common/views/components/Body/Body.tsx

```diff
@@ -281,16 +290,21 @@ const Body: FunctionComponent<Props> = ({
           {slice.type === 'inPageAnchor' && <span id={slice.value} />}
           {/* If the first slice is featured text we display it above inPageAnchors and any static content, i.e. <AdditionalContent /> */}
           {i === 0 && slice.type === 'text' && slice.weight === 'featured' && (
-            <LayoutWidth width={minWidth}>
+            <Layout8 shift={!sectionLevelPage && true}>
```

Comment:
shift = true moves the layout with padding

Reply:
sure, but the `&& true` specifically? Looks weird, but I'm guessing it might be a short-circuit. Would it not work as `!sectionLevelpage` alone (because `sectionLevelPage` could be undefined I guess)?

Reply:
layout shift is default to true in the fc itself. we could rename this to something else? but i wanted to keep the default behavior